tracker/settings/base.py

```diff
@@ -251,6 +254,45 @@
 # django-taggit
 TAGGIT_CASE_INSENSITIVE = True
 
+# Content Security Policy
+CSP_DEFAULT_SRC = ("'self'",)
+CSP_SCRIPT_SRC = (
+    "'self'",
+    "https://analytics.freedom.press",
+    # For Wagtail Admin
+    "'unsafe-inline'",
+    # For Twitter Widgets
+    "'unsafe-eval'",
+    "https://platform.twitter.com/widgets.js",
+    "https://platform.twitter.com/js/horizon_tweet.2bd42981e3af03ce9186a5655508da28.js",
+)
+CSP_STYLE_SRC = (
+    "'self'",
+    # For Wagtail Admin and Twitter Widgets.
+    "'unsafe-inline'",
+)
+CSP_FRAME_SRC = (
+    "'self'",
+    # For Twitter Widgets
+    "https://platform.twitter.com",
+)
+CSP_CONNECT_SRC = (
+    "'self'",
+    "https://analytics.freedom.press",
+    # For Wagtail Admin
+    "https://releases.wagtail.io/latest.txt",
+)
+CSP_IMG_SRC = (
+    "'self'",
+    "analytics.freedom.press",
+    "https://media.pressfreedomtracker.us",
+    # For Wagtail Admin
+    "http://www.gravatar.com",
+)
```

Comment:
With previous experience in other repos, I think you will need:

```
# Embeds
    'https://platform.twitter.com',  # Twitter
    'https://syndication.twitter.com',  # Twitter
    'https://pbs.twimg.com',  # Twitter
    'https://ton.twimg.com',  # Twitter
    'data:',  # Twitter makes use of data svgs
```
to be added to CSP_IMG_SRC to allow twitter images.